oggm_edu/funcs.py

```diff
@@ -86,37 +115,38 @@ def glacier_plot(x, bed, model, mb_model, init_flowline):
 
 
 def init_model(init_flowline, mb_model, years, glen_a=None, fs=None):
-    """This function initializes a new model, therefore it uses FlowlineModel.
+    """This function initializes a new model, therefore it uses FluxBasedModel.
     The outline of the glacier is calculated for a chosen amount of years.
 
-    TODO: return also length and volume.
-
     Parameters
     ----------
-    init_flowline : flowline
-    mb_model : mass balance model
+    init_flowline : oggm.core.flowline.RectangularBedFlowline
+        the glacier flowline
+    mb_model : oggm.core.massbalance.LinearMassBalance
+        the glacier mass balance model
     years : int
-        glacier will be calculated for the chosen amount of years
-    glen_a : float
-        Glen's parameter (default = 2.4e-24)
-    fs : float
-        sliding parameter (default = 0)
+        year until which glacier evolution is calculated
+    glen_a : float, optional
+        Glen's parameter, (default: 2.4e-24 s^-1 Pa^-3)
+    fs : float, optional
+        sliding parameter, (default: 0)
 
    Returns
    -------
-   model : flowline.FluxbasedModel
+   model : oggm.core.flowline.FluxBasedModel
+       the initialized model
 
-   TODO: return also length and volume steps (they are for every time step
-   calculated)
-    """
-    if glen_a is None:
+   TODO: return also length and volume steps (they are calculated for every
+         time step)
+   """
+    if not glen_a:
```

Comment:
Note that the pythonic way here indeed is `if glen_a is None:`. The reason is that the kwarg value of 0 could be a valid entry (in this case it's physically not possible, but in many cases it is). 

No need to change it, I'll do it for you now